aws/internal/service/s3/errors.go

```diff
@@ -0,0 +1,5 @@
+package s3
+
+const (
+	NoSuchTagSet = "NoSuchTagSet"
```

Comment:
For the future, we should likely prefix these with `ErrCode`, similar to the AWS Go SDK.

Reply:
Looking at `aws/internal/service/ec2/errors.go` there is currently some inconsistency so a stated position in the Contributing Guide will be helpful.

Reply:
Opened https://github.com/terraform-providers/terraform-provider-aws/pull/15516 to attempt to document this.